src/main/java/ddingdong/ddingdongBE/domain/form/service/FacadeCentralFormServiceImpl.java

```diff
@@ -254,12 +254,14 @@ private void validateEndDate(LocalDate startDate, LocalDate endDate) {
     private List<FormField> toUpdateFormFields(Form originform,
                                                List<UpdateFormFieldCommand> updateFormFieldCommands) {
         return updateFormFieldCommands.stream()
-                .map(formFieldCommand -> formFieldCommand.toEntity(originform)).toList();
+                .map(formFieldCommand -> formFieldCommand.toEntity(originform))
+                .toList();
     }
 
     private List<FormField> toCreateFormFields(Form savedForm,
-                                               List<CreateFormFieldCommand> createFormFieldCommands) {
+            List<CreateFormFieldCommand> createFormFieldCommands) {
         return createFormFieldCommands.stream()
-                .map(formFieldCommand -> formFieldCommand.toEntity(savedForm)).toList();
+                .map(formFieldCommand -> formFieldCommand.toEntity(savedForm))
+                .toList();
```

Comment:
p4)
```suggestion
                .map(formFieldCommand -> formFieldCommand.toEntity(savedForm))
                ..flatMap(formField -> formField.generateFormFieldsBySection(form))
                .toList();
```
createForm() 메서드의 스트림과 합쳐도 괜찮을거 같습니다!

Reply:
헙 그러네요 수정하였습니다!